strapi/strapi
# Helper-Plugin: Replace babel with esbuild

### What does it do?

This is a first POC to replace babel with esbuild for the helper-plugin. It:

- reduces the build-time for the plugin from ~8s to 2s
- reduces the time spent in `yarn setup` from ~24s to 🥁 ~19s
- reduces the bundle-size by about ~80kB
- reduces dependencies and therefore installation size and time

> **Note**
> The snapshot updates shuffle mainly styles around. I haven't seen any actual difference.

### Why is it needed?

We'd like to maintain a consistent build-pipeline across the project. Bundle and installation-size are essential metrics.

### How to test it?

Run a fresh build in one of the examples (without a `build` and `.cache` directory) and verify everything still works.
